docs/DEVELOPING.md

```diff
@@ -1,5 +1,187 @@
 # Connectors Developer's Guide
 
+## Network Drives Connector
+
+Elastic Network Drives connector is provided in Elastic Connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
```

Comment:
Let's link "Network Drives connector" above to /connectors/sources/network_drive.py
and similarly for S3 to s3.py.

Reply:
I see the link goes to main. We should make sure the link references the 8.6 source when it's backported to 8.6.

Reply:
> I see the link goes to main. We should make sure the link references the 8.6 source when it's backported to 8.6.

The code in 8.6 branch in not updated. Previously for s3 we had aws.py file but now it is renamed to s3.py. So as of now is it fine if we can use the main branch link? Once the code is backported to 8.6 we'll update the document to use the 8.6 branch.

Reply:
That's fine, as long you make sure the 8.6 branch has the correct links.